lib/tasks/xsd_change_log.rake

```diff
@@ -38,6 +38,33 @@ namespace :xsd do
       build_diff(dataset, dv1, dv2, print_output)
     end
 
+    desc 'v8/v9 differences'
+    task cosd_v8_v9_diffs_rawtext: :environment do
+      filename = ENV['FILE']
+      require 'csv'
+      rows = []
+      CSV.open(filename, "r+") do |csv|
+        csv.each_with_index do |row, i|
+          next if i == 0
+          rows << [row[0], row[1], row[2], row[3], row[4], row[5], row[6]]
```

Comment:
and then `rows.to_h.values `   ?

Reply:
2 columns have the same header so that the user can interpret the similarities